app/http/routes.js

```diff
@@ -239,7 +238,6 @@ module.exports = function (http, modelsController, webmakerAuth) {
     "/api/v2/user/request-reset-code",
     authMiddleware,
     routes.user3.setUser(modelsController),
-    routes.user3.doesUserHavePassword(true),
```

Comment:
note: tested that removing this line and 233 allows resets on accounts that don't have passwords set (which is the use case we need to support)